agot-bg-game-server/src/common/ingame-game-state/game-data-structure/house-card/TheBlackfishHouseCardAbility.ts

```diff
@@ -4,8 +4,7 @@ import House from "../House";
 import HouseCard from "./HouseCard";
 
 export default class TheBlackfishHouseCardAbility extends HouseCardAbility {
-
-    doesPreventCasualties(_combat: CombatGameState, _house: House, _houseCard: HouseCard): boolean {
-        return true;
+    doesPreventCasualties(_combat: CombatGameState, _house: House, houseCard: HouseCard, affectedHouseCard: HouseCard): boolean {
```

Comment:
@Longwelwind After searching a solution for the final combat strength problem I realized that this method is not implemented in a way that another house card could prevent casualties as well. The thing is, it is no problem as Blackfish itself has sword icons and cannot generate some. But the current implementation will prevent casualties for both combatants when Blackfish is drawn (We had the same already with Catelyn, she doubled the defense bonus of the opponent as well). So I fixed that as well. There may be other abilities as well with a similar problem...